### PR TITLE
fix(DB/gameobject): Archbishop Alonsus Faol misplaced plaque in Stormwind

### PR DESCRIPTION
**Pull request by WojtekNosal**
## Changes Proposed:
-  Placing Archbishop Alonsus Faol plaque in the right place

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4642

## Tests Performed:
- Tested in-game now plaque is correctly positioned
WoWScrnShot_031521_035930
WoWScrnShot_031521_035918

## How to Test the Changes:
- .go gobject 26628
- Check Archbishop Alonsus Faol plaque placed on the fountain

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by markvetx** [comment]
Hi! According to [this](https://www.wowhead.com/object=177199/archbishop-alonsus-faol#screenshots) it should be placed on it's back to the ground, so it should face upwards. 

It looks okay on vmangos tho, but have an identical position values in 'gameobject' table

**Comment by Si1ker** [comment]
> Hi! According to [this](https://www.wowhead.com/object=177199/archbishop-alonsus-faol#screenshots) it should be placed on it's back to the ground, so it should face upwards.
> 
> It looks okay on vmangos tho, but have an identical position values in 'gameobject' table

Correct.

image

Upwards :)

**Comment by WojtekNosal** [comment]
Ohh sorry I've checked WoWHead for plaque placement but tought it should be like that. Thanks guys for pointing me out the correct plaque position, I will commit fix later today, cheers!

**Comment by WojtekNosal** [comment]
Plaque is now correctly positioned
WoWScrnShot_031721_220650

File: data/sql/updates/pending_db_world/rev_1615775571176561100.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1615775571176561100');
+
+DELETE FROM `gameobject_addon` WHERE `guid` = 26628;
+INSERT INTO `gameobject_addon` VALUES (26628, 0, 0);
+UPDATE `gameobject` SET `position_x` = '-8640.98', `position_y` = '760', `position_z` = '98.38' WHERE `guid` = 26628;
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1615775571176561100.sql:5]:
```suggestion
UPDATE `gameobject` SET `position_x` = -8640.98, `position_y` = 760, `position_z` = 98.38 WHERE `guid` = 26628;
```

**Review by Si1ker — CHANGES_REQUESTED** [review]
Numbers are not strings, avoid using  ' '  if you are not changing auras.

**Review by Si1ker — APPROVED** [review]
Tested. ✔️ 

/* Affected rows: 2  Found rows: 0  Warnings: 0  Duration for 3 queries: 0.000 sec. */

image

**Review by Voxstrasza — APPROVED** [review]

**Pull request merged**